kolibri/core/auth/api.py

```diff
@@ -180,6 +183,19 @@ class FacilityUserViewSet(ValuesViewset):
         "is_superuser": lambda x: bool(x.pop("devicepermissions__is_superuser"))
     }
 
+    @list_route(methods=["get"])
+    def users_for_facilities(self, request):
+        facility_ids = dict(request.GET)["member_of"]
+
+        # The GET params come as an array with one comma-separated str if
+        # there are more than 1 facility_id being requested
+        if "," in facility_ids[0]:
+            facility_ids = facility_ids[0].split(",")
+
+        users = FacilityUser.objects.filter(facility__in=facility_ids)
+        serialized = FacilityUsersForFacilitiesSerializer(users)
```

Comment:
When I profiled this query on my local machine, it ran 900+ queries, and returned a JSON string (not an object) - no need to double render the data in the line below - the Response does that internally.